core/src/main/java/org/apache/any23/rdf/RDFUtils.java

```diff
@@ -528,10 +528,10 @@ public static boolean isAbsoluteIRI(String href) {
             new java.net.URI(href.trim());
```

Comment:
On an unrelated note, why are we testing this on `href.trim()` instead of just `href`? 
`RDFUtils.iri(href)` does *not* call `href.trim()`!